toonz/sources/toonzlib/preferences.cpp

```diff
@@ -625,6 +631,17 @@ Preferences::Preferences()
   getValue(*m_settings, "watchFileSystemEnabled", m_watchFileSystem);
   getValue(*m_settings, "shortcutCommandsWhileRenamingCellEnabled",
            m_shortcutCommandsWhileRenamingCellEnabled);
+
+  QString xsheetLayoutPreference;
+  xsheetLayoutPreference =
+      m_settings->value("xsheetLayoutPreference").toString();
+  if (xsheetLayoutPreference != "")
+    m_xsheetLayoutPreference = xsheetLayoutPreference;
+  else if (existingUser)  // Existing users with missing preference defaults to
+                          // Classic. New users will be Compact
```

Comment:
It's just a personal opinion - I think the "Classic-revised" layout would be good for new users.
I suppose that the "Compact" layout would be more for experienced user who already know the purpose of each button and would like to gain more working space in the cell area.
And IMO it would be more friendly to offer larger buttons available in the "Classic-revised" (or "Classic") layout for new users, considering that it would be more preferable for them to get familiar with usage of the buttons than to create long footage.

Reply:
I agree with that as well. I've switch the default for new users to Classic-revised.

Reply:
(also personal opinion here)
I just find the Compact layout more easy to new users because of being more:
- Simple: all buttons are on a single line.
- Coherent: no matter if you use xsheet or timeline modes, you always find this buttons in a similar layout (i.e. single line, same left to right order)
- Compatible: it's pretty much how Toonz has them also.
(NOTE: although I"ve just realized we have the CamStand and Render buttons order swapped with respect to how Tonnz shows them). Why that happened?? :-\   Could it be inverted before it goes "official"? ...it would be easier for users having to use both programs). @manongjohn @turtletooth @shun-iwasawa @konero

Reply:
@gab3d 
Thank you for the suggestion. I understand that the UI-related issue is really a matter of preference and the best answer would vary among users. But I still think that the "Classic-revised" is more for new-users than "Compact" ...
The preview and camstand toggle buttons are used IMO more frequently than the other two. I think that having them larger will afford users such importance. 
Considering the consistency with other software, the "eye" shaped button is commonly recognized as some visible property, but the "animation table" icon is uncommon and quite original in OpenToonz. Making the "preview" and "camstand" buttons the same (large) size will also help to know that both buttons have similar functionalities.

> (NOTE: although I"ve just realized we have the CamStand and Render buttons order swapped with respect to how Toonz shows them). Why that happened?? :-\ Could it be inverted before it goes "official"?

Thank you for pointing the inconsistency in order of icons with Harlequin, I didn't realize that. 
I originally made the current column head layout based on Toonz 4.6 which Ghibli staffs had been already familiar with. In Toonz 4.6 there was no concept of camstand visibility, so I just added it under the preview toggle.
<img src="https://user-images.githubusercontent.com/17974955/31158227-42da896c-a8fb-11e7-843e-97fbe61c649f.png" width=450>

I think having the preview toggle at the first place is somewhat reasonable even if it will lose some consistency with Harlequin, considering that the preview toggle is important as it actually affect to the render result.